### DISCUSSION TITLE
A record

### DISCUSSION THREAD

**Discussion by vaimuaare** [community/community#101642] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

For which IP address the A record for my domain should be directed?

Only CNAME is not enough.

**Comment by fury-05** [comment]
Can you give more details regarding which ip you are talking about?

**Reply by varje** [reply]
My web host company wants GitHub IP address to set up record A corresponding to my web address.

**Reply by fury-05** [reply]
You can find documentation about a record configuration [here ](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#:~:text=ANAME%2C%20or-,A%20record,-.%20You%20can%20also)
